.circleci/config.yml

```diff
@@ -7,6 +7,7 @@ workflows:
       - test-3.6
       - test-3.7
       - test-3.8
+      - test-latest
```

Comment:
Shouldn't we explicitly list the supported version, e.g. `test-3.9`?

Reply:
I think we should review the python 3.9 change log, add a `test-3.9` specific test, and update docs to officially support 3.9. But the purpose of `test-latest` could be to potentially find code changes that aren't forward compatible and require further investigation. Or could just be annoying because it fails due to something you don't have time to deal with.